articles/active-directory/hybrid/plan-connect-userprincipalname.md

```diff
@@ -76,6 +76,8 @@ When the updates to a user object are synchronized to the Azure AD Tenant, Azure
 >Azure AD recalculates the UserPrincipalName attribute value only in case an update to the on-premises UserPrincipalName attribute/Alternate login ID value is synchronized to the Azure AD Tenant. 
 >
 >Whenever Azure AD recalculates the UserPrincipalName attribute, it also recalculates the MOERA. 
+>
+>In case of verified domain change, Azure AD also recalculates the UserPrincipalName attribute. For more information, see [Troubleshoot: Audit data on verified domain change](https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/troubleshoot-audit-data-verified-domain)
```

Comment:
```suggestion
>In case of verified domain change, Azure AD also recalculates the UserPrincipalName attribute. For more information, see [Troubleshoot: Audit data on verified domain change](https://docs.microsoft.com/azure/active-directory/reports-monitoring/troubleshoot-audit-data-verified-domain)
```